testsuite/integration-arquillian/tests/base/testsuites/base-suite

```diff
@@ -45,3 +45,4 @@ validation,6
 vault,4
 welcomepage,6
 x509,4
+oid4vc, 6
```

Comment:
Probably should not have a space and should be in correct line according to alphabetical order (I suppose CI script fails due to this).

Reply:
Fixed in [272066b](https://github.com/keycloak/keycloak/pull/26484/commits/272066b92015fcd415c9f5c5af75fa34e2c66dc5)